composer.json

```diff
@@ -41,7 +41,9 @@
         "sentry/sentry": "^2.4",
         "sly/notification-pusher": "2.x",
         "vlucas/phpdotenv": "^2.4",
-        "bakaphp/php-apple-signin": "^2.0"
+        "bakaphp/php-apple-signin": "^2.0",
+        "facebook/graph-sdk": "5.7.0",
```

Comment:
@FredPeal is there no new fb library ? this is 2 years old

Reply:
Yes, but this is the lasted version for PHP.
For the validation is okay.